src/lib.rs

```diff
@@ -29,7 +29,14 @@
 //!        .buffer_unordered(1usize);
 //!
 //!    while let Some(_handle_message) = handlers.next().await {}
-//!
+//!    drop(handlers); // close the channel so the sender will stop
```

Comment:
Am I right that it will be done automatically when we exit from the current function?